opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -200,3 +201,80 @@ def collect(self) -> Optional[Histogram]:
             aggregation_temporality=AggregationTemporality.DELTA,
             sum=self._sum,
         )
+
+
+def _convert_aggregation_temporality(
+    previous_point: Optional[_PointVarT],
+    current_point: _PointVarT,
+    aggregation_temporality: AggregationTemporality,
```

Comment:
```suggestion
    aggregation_temporality: AggregationTemporality,
    current_point: _PointVarT,
    previous_point: Optional[_PointVarT] = None,
```

Reply:
:thinking: but why? doesn't it make more sense to have `previous_point` before `current_point`?

Reply:
No strong opinion, just a minor suggestion based on the fact that `previous_point` could be optional.

Reply:
Oh, I would have to update all the test cases :sweat_smile: reverting... also, I think this function is always being called with two points (even if one of them is `None`).